<?xml version="1.0"?>
<launch>

    <!-- X Y Z Y P R transform of marvel origin frame wrt map frame. Needs to be physically measured. -->
    <arg name="origin_pos" default="1.52 -2.37 0 0.55 0 0"/>

    <!-- Publish static TF of the origin anchor to map -->
    <node pkg="tf" type="static_transform_publisher" name="marvel_tf" args=" $(arg origin_pos)  /map /marvel_origin 1"/>

    <!-- <node pkg="marvelmind_nav" type="hedge_rcv_bin" name="hedge_rcv_bin" output="screen" /> -->

    <!-- <node pkg="araig_interpreters" type="marvel_location_interpreter" name="marvel_interpreter" output="screen" /> -->
    <node pkg="araig_interpreters" type="marvel_location_interpreter" name="marvel_interpreter" output="screen"> -->
        <remap from="/marvel_interpreter/beacon10" to="/data/interpreter/location/robot"/>
        <remap from="/marvel_interpreter/beacon11" to="/data/interpreter/location/obstacle"/>
    </node>
</launch>
